NitroxServer/Communication/Packets/Processors/PlayerJoiningMultiplayerSessionProcessor.cs

```diff
@@ -33,36 +35,44 @@ public override void Process(PlayerJoiningMultiplayerSession packet, NitroxConne
 
             Optional<EscapePodModel> newlyCreatedEscapePod;
             NitroxId assignedEscapePodId = world.EscapePodManager.AssignPlayerToEscapePod(player.Id, out newlyCreatedEscapePod);
-            if(newlyCreatedEscapePod.IsPresent())
+            if (newlyCreatedEscapePod.IsPresent())
             {
                 AddEscapePod addEscapePod = new AddEscapePod(newlyCreatedEscapePod.Get());
                 playerManager.SendPacketToOtherPlayers(addEscapePod, player);
             }
 
             List<EquippedItemData> equippedItems = world.PlayerData.GetEquippedItemsForInitialSync(player.Name);
-            List<NitroxModel.DataStructures.TechType> techTypes = equippedItems.Select(equippedItem => equippedItem.TechType).ToList();
+            List<TechType> techTypes = equippedItems.Select(equippedItem => equippedItem.TechType).ToList();
 
             PlayerJoinedMultiplayerSession playerJoinedPacket = new PlayerJoinedMultiplayerSession(player.PlayerContext, techTypes);
             playerManager.SendPacketToOtherPlayers(playerJoinedPacket, player);
 
+            // Make players on localhost admin by default.
+            if (IPAddress.IsLoopback(connection.Endpoint.Address))
+            {
+                world.PlayerData.SetPermissions(player.Name, Perms.ADMIN);
+            }
+            Perms initialPerms = world.PlayerData.GetPermissions(player.Name);
+            Log.Info($"Changed player '{player.Name}' on '{connection.Endpoint.Address}' permissions to '{initialPerms.ToString()}'");
```

Comment:
Do we always want to log?

Reply:
In my opinion, this would just be a given when you host the server so I would say we probably don't need to log it at all

Reply:
Sounds good to me.